bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+foreman_root = '/usr/share/foreman'
```

Comment:
I can do it but is it worth it? We haven't changed the target installation directory ever and we've relied on this path on foreman-tasks forever